### ISSUE TITLE
update blocks without list aren't printed correctly

### ISSUE THREAD

**Issue by arr2036** [FreeRADIUS/freeradius-server#1473]
```
update {
	&reply: += &request:[*]
}
```

```
(0)                :      {
(0)                :       &reply::User-Name += &User-Name:0[*] -> 'foo'
(0)                :       &reply::User-Password += &User-Password:0[*] -> 'bar'
(0)                :       &reply::NAS-IP-Address += &NAS-IP-Address:0[*] -> 127.0.0.1
(0)                :       &reply::NAS-Port += &NAS-Port:0[*] -> 0
(0)                :       &reply::Message-Authenticator += &Message-Authenticator:0[*] -> 0x57649de50a481d9a93cee755d71f4b0e
(0)                :     } #  (noop)
```

**Comment by alandekok** [comment]
You sure this wasn't fixed by commit 9c5539c0596e ?

**Comment by arr2036** [comment]
Ah yes, hadn't pulled in a while.  Sorry for the noise.
